src/main/java/ai/ancf/lmos/operator/service/AgentServiceQuery.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Deutsche Telekom AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package ai.ancf.lmos.operator.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.MediaType;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@org.springframework.stereotype.Service
+public class AgentServiceQuery {
+
+    private static final Logger LOG = LoggerFactory.getLogger(AgentServiceQuery.class);
+
+    private final WebClient webClient;
+
+    public AgentServiceQuery() {
+        this.webClient = WebClient.builder().build();
+    }
+
+    public String queryAgentService(String serviceUrl) {
```

Comment:
The URI Path should also be dynamic.
Usually we would say that the path should be `.well-known/wot` by default, which is specified by [WoT Discovery](https://www.w3.org/TR/wot-discovery/#introduction-well-known). 

But let's say the discovery endpoint is different, e.g. `agent`, it could be specified in the deployment file:

```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: agent-deployment
  labels:
    app: agent-deployment
    wot-agent: "true"   # Signal that it's an Agent which is deployed
  annotations:
    wot/td-endpoint: "/agent"  # Custom endpoint of the agent thing description
```

So we should check if the annotations is available in the Deployment, if yes, take it, otherwhise use the default endpoint. Clear what I mean?